frontend/libretro.c

```diff
@@ -31,7 +31,7 @@
 #include "plugin_lib.h"
 #include "arm_features.h"
 #include "revision.h"
-#include "libretro.h"
+#include "libretro-common/include/libretro.h"
```

Comment:
A cleaner solution would be to add `libretro-common/include` to the include directories in Makefile.common. Then just use `#include "libretro.h"`, and the compiler will file it correctly.